test/acceptance/js/LearnTest.js

```diff
@@ -51,3 +59,24 @@ describe('learning words', () => {
     expect(responseBody.misspellings.length).to.equals(1)
   })
 })
+
+describe('unlearning words', () => {
+  it('should return status 200 when posting a word successfully', async () => {
+    const response = await unlearnWord('anything')
+    expect(response.statusCode).to.equal(200)
```

Comment:
See my comment above regarding a changed status code.
```suggestion
    expect(response.statusCode).to.equal(204)
```